packages/block-library/src/quote/v2/block.json

```diff
@@ -0,0 +1,19 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "core/quote",
+	"title": "Quote",
+	"category": "text",
+	"description": "Give quoted text visual emphasis. \"In quoting others, we cite ourselves.\" — Julio Cortázar",
+	"keywords": [ "blockquote", "cite" ],
+	"textdomain": "default",
+	"attributes": {
+		"attribution": {
```

Comment:
Why not just reuse the same attribute name as the existing quote "citation"?

Reply:
(reasoning being to make the least possible breaking changes)

Reply:
The problem is that it's a breaking change for folks using quotes in "CPT templates". That said, I think the problem will still exist for folks using the "value" attribute which now becomes inner blocks. So would be good to provide a "migration" for these templates. We don't have "migrations" at the framework level but maybe we can start adhoc:

 - Leave the old and new attributes
 - if we detect a block with the old attributes, use a `useEffect` or something to migrate the attributes.

The ideal scenario would be to implement "migrations" in addition to "deprecations".

Reply:
Even if we maintain the name, the selector has changed from `cite` to `figcaption`, so it's going to be a breaking change for anyway.

Reply:
Oh, thanks for the clarification.